otlp/traces.go

```diff
@@ -13,9 +14,11 @@ import (
 )
 
 const (
-	traceIDShortLength = 8
-	traceIDLongLength  = 16
-	defaultSampleRate  = int32(1)
+	traceIDShortLength     = 8
+	traceIDLongLength      = 16
+	traceIDCorruptedLength = 24
+	spanIDCorruptedLength  = 12
```

Comment:
Could we rename these? Trace & span IDs aren't corrupted, they just aren't base64 encoded like when receiving via OTLP/protobuf.

Reply:
👍